eventsapi/main.py

```diff
@@ -0,0 +1,28 @@
+import os
+import logging
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from eventsapi import routers
+
+logging.basicConfig(level=logging.INFO)
+
+app = FastAPI(
+    title="RAISE Events API"
+)
+
+app.include_router(routers.v1_router, prefix="/v1")
```

Comment:
I'm curious about the decision to use this fastapi router? Is the idea that we can have different 'versions' of the API, and the routers allow the same `/events` endpoint to get used despite the prefix like `/v1/events`, `/v2/events`. How and where then are v1/v2 differences distinguished? Does this help with backwards compatibility?

Reply:
Yeah, the use of the version is per our [documentation](https://docs.google.com/document/d/1FataAirgaqFUO2W4S06njnKGPgIuRIKP6Isyct3s_zM/edit#) so we can support things like `/v2` if ever needed in the future. The use of the router is mostly a nicety at this stage to avoid having to encode `v1/events` explicitly. I just find it cleaner to separate the endpoint code from the main per [these docs](https://fastapi.tiangolo.com/tutorial/bigger-applications/?h=apirouter#another-module-with-apirouter), but it is a bit of subjective choice at this stage since there's very little actual code :man_shrugging: .